### DISCUSSION TITLE
Random playlist generator

### DISCUSSION THREAD

**Discussion by tommyp-vr4** [gerbera/gerbera#1632] [General]
Is there a way to add a random playlist generator [container] with say X songs?  I've been using my controller to add "everything" to the player and then make it play random.  That just makes a large random playlist which is good, but sometimes the controller gets all confused and times out if I send too many songs to the player's playlist.

**Comment by KarlStraussberger** [comment]
It cannot be really random, because once you click the container has to generate a playlist of X items from, e.g. 50000, entries in your database (what's the "random" criteria?). To make it playable it has to stay there for a while. This means we have to generate a temporary item which is dropped after the grace period. This may work if the client stores the playlist and if you open it again after you reached the end.
